### PR TITLE
[RFR] TODO consistency 

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- It will help to avoid confusion. 
- Lets use same `TODO` style for overall repo.
- Added small point in our guide

Reference:
code tags: https://www.python.org/dev/peps/pep-0350/#id30
Google code style: http://google.github.io/styleguide/pyguide.html

{{pytest: cfme/tests/cloud/test_tenant.py::test_tenant_crud -v}}

### CODE REVIEW

File: docs/guides/dev_guide.rst [hunk 1]
```diff
@@ -249,6 +249,16 @@ We also do a few things that aren't explicitly called out in PEP 8:
           CertainQuestionsError: Raises certain questions about the authors sanity.
         """
 
+* Use `TODO` comment for workaround, temporary or not perfect code.
+  A `TODO` comment begins with the string `TODO` all in caps. It can use with some identifier like
```
Comment (john-dupuy) [inline docs/guides/dev_guide.rst]: Change "It can use with ..." to "It can be used with"

File: docs/guides/dev_guide.rst [hunk 2]
```diff
@@ -249,6 +249,16 @@ We also do a few things that aren't explicitly called out in PEP 8:
...
+  Name or Email (responsible for improvement), Bugzilla, Github Issue etc.
```
Comment (john-dupuy) [inline docs/guides/dev_guide.rst]: Change `(responsible for improvement)` to `(i.e. the person responsible for improvement)`

File: docs/guides/dev_guide.rst [hunk 3]
```diff
@@ -249,6 +249,16 @@ We also do a few things that aren't explicitly called out in PEP 8:
...
+  An example is described below::
```
Comment (john-dupuy) [inline docs/guides/dev_guide.rst]: Fix typo with `::`, change to `:`

Comment (john-dupuy) [reply docs/guides/dev_guide.rst]: Optional: this can just read `For example:`

File: cfme/services/catalogs/__init__.py [hunk 4]
```diff
@@ -53,8 +53,8 @@ class toolbar(View):  # noqa
         policy = Dropdown('Policy')
         view_selector = View.nested(ItemsToolBarViewSelector)
 
-    # for backward compatibility. it is difficult to figure out where those are used
-    # TODO: this should be fixed by this code owner
+    # TODO(nansari/sshveta): this should be fixed by this code owner
```
Comment (john-dupuy) [inline cfme/services/catalogs/__init__.py]: Change `fixed by this code owner` to `fixed by the indicated code owner`

File: cfme/services/catalogs/__init__.py [hunk 5]
```diff
@@ -53,8 +53,8 @@ class toolbar(View):  # noqa
...
+    #  for backward compatibility.it is difficult to figure out where those are used
```
Comment (john-dupuy) [inline cfme/services/catalogs/__init__.py]: Fix the typo at `compatibility.it`

File: cfme/cloud/tenant.py [hunk 6]
```diff
@@ -313,16 +312,10 @@ def create(self, name, provider, wait=True):
         else:
             page.form.cancel_button.click()
 
-        all_view = self.create_view(TenantAllView)
-        wait_for(lambda: all_view.is_displayed, num_sec=120, delay=3,
-                 fail_func=all_view.flush_widget_cache, handle_exception=True)
+        all_view = self.create_view(TenantAllView, wait="20s")
```
Comment (john-dupuy) [inline cfme/cloud/tenant.py]: @digitronik Is this related to the `TODO` chage?

File: docs/guides/dev_guide.rst (hunk 3 above)
Comment (digitronik) [reply docs/guides/dev_guide.rst]: It's not typo.

File: cfme/services/catalogs/__init__.py (hunk 5 above)
Comment (digitronik) [reply cfme/services/catalogs/__init__.py]: about space...

File: cfme/cloud/tenant.py (hunk 6 above)
Comment (digitronik) [reply cfme/cloud/tenant.py]: No but I would like to put in same PRs. While working on one PR; If i found old implementation and no need to run test then I introduce in same PR to avoid multiple PRs.

File: cfme/services/catalogs/__init__.py (hunk 4 above)
Comment (digitronik) [reply cfme/services/catalogs/__init__.py]: I think better i remove this like `identifier` itself indicating owners.

**Comment by digitronik** [comment]
> looks neat.
> @digitronik, please add some tenant test to PRT to check that you change won't break something.
It was my bad... I added test run :+1:
